### PR TITLE
ECM-128: Move the UI to the Front-end

### PR DESCRIPTION
**Pull request by JoseHawk**

### CODE REVIEW

File: services/educama-supplier-simulator-frontend/src/app/event-list/event-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import { Flight } from '../models/flight';
+import { Cargo } from '../models/cargo';
+import { FlightListService } from './flight-list/flight-list.service';
+import { CargoListService } from './cargo-list/cargo-list.service';
+
```
Comment (larsduelfer) [inline services/educama-supplier-simulator-frontend/src/app/event-list/event-list.component.ts]: Remove unused imports.

File: services/educama-supplier-simulator-frontend/src/app/event-form/event-form.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (larsduelfer) [inline services/educama-supplier-simulator-frontend/src/app/event-form/event-form.component.ts]: Remove unused imports.

File: services/educama-supplier-simulator-frontend/src/app/app.module.ts [hunk 3]
```diff
@@ -2,28 +2,53 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
+import { RouterModule, Routes } from '@angular/router';
```
Comment (larsduelfer) [inline services/educama-supplier-simulator-frontend/src/app/app.module.ts]: Remove unused imports.

File: services/educama-supplier-simulator-frontend/src/app/event-form/cargo-form/cargo-form.service.ts [hunk 4]
```diff
@@ -2,15 +2,15 @@ import { Injectable } from '@angular/core';
 import { Http } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
 
-import { CargoForm } from './cargo-form';
+import { Cargo } from '../../models/cargo';
 
 @Injectable()
```
Comment (larsduelfer) [inline services/educama-supplier-simulator-frontend/src/app/event-form/cargo-form/cargo-form.service.ts]: The url's must NOT be hard coded here. How shall we find the right url, if we deploy the app to the DEV stage?
We have a solution for this in the educama frontend. Please have a look there.

File: services/educama-supplier-simulator-frontend/src/app/event-form/flight-form/flight-form.service.ts [hunk 5]
```diff
@@ -2,15 +2,15 @@ import { Injectable } from '@angular/core';
 import { Http } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
 
-import { FlightForm } from './flight-form';
+import { Flight } from '../../models/flight';
```
Comment (larsduelfer) [inline services/educama-supplier-simulator-frontend/src/app/event-form/flight-form/flight-form.service.ts]: Please do NOT hardcode the URL here. See comment for CargoFormService.

**Comment by larsduelfer** [comment]
Please also remove the link to the old form (which was implemented in the backend). It is not valid anymore. (See test deployment on sandbox)

File: services/educama-supplier-simulator-frontend/src/app/app.module.ts (hunk 3 above)
Comment (JoseHawk) [reply services/educama-supplier-simulator-frontend/src/app/app.module.ts]: Both are being used:

const appRoutes: Routes = [
  { path: '', component: EventListComponent },
  { path: 'form', component: EventFormComponent }
];

 imports: [
    RouterModule.forRoot(
      appRoutes,
      { enableTracing: true }
    ),
    BrowserModule,
    FormsModule,
    HttpModule
  ],

**Pull request closed without merging**